plenum/test/consensus/test_consensus_data_provider.py

```diff
@@ -23,5 +27,31 @@ def test_initial_consensus_state(some_item, other_item, validators):
     assert data.prepared == []
 
     # Checkpoints
-    assert data.stable_checkpoint == 0
-    assert data.checkpoints == []
+    assert data.stable_checkpoint == None
+    assert list(data.checkpoints) == []
+
+
+def test_pp_storages(pre_prepare, provider):
```

Comment:
I'd prefer to split this test into separate ones testing different scenarios

Reply:
ok